packages/chats/src/BubblePlainText.tsx

```diff
@@ -153,24 +161,26 @@ function PlainTextContainer(
     const roundedCornerStyle = useBubbleRoundedCornerStyle(props, position);
     const actionString = getActionString(props);
 
+    const textLongPressHandler = useTextLongPressHandler()
+
+    const copyToClipboard = React.useCallback(()=>{
+        bubbleScaleAnimation(true);
+        props.text && Clipboard.setString(props.text);
+        textLongPressHandler && textLongPressHandler()
```

Comment:
Not sure if we need to copy the text just right here (why not doing it from the parent?)
My thoughts is that you won't understand that the text was copied after the long press (since no toast has been shown yet about the fact it was copied...)

@savelichalex what do you think?

Also, I do think that we need to pass the text itself to the `textLongPressHandler` (as we do it for URL to process it in the parent)